src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.spec.ts

```diff
@@ -249,6 +256,60 @@ describe('PaypalCommercePaymentProcessor', () => {
         });
     });
 
+    describe('render apmFields', () => {
+        it('setting PaypalCommerce apmFields without styles and fields predifined values', async () => {
```

Comment:
for test names, it's good to read them some as a sentence, e.g.
`it('sets PaypalCommerce')` as in `it sets PaypalCommerce`